admin/blockly.js

```diff
@@ -337,6 +351,7 @@ Blockly.JavaScript['telegram_call'] = function(block) {
         '\n    text: ' + value_message +
         (value_username ? ',\n    ' + 'user: ' + value_username : '') +
         ',\n    lang: "' + dropdown_language + '"' +
+        ',\n    repeats: "' + repeats + '"' +

```

Comment:
is repeats really a String?

Reply:
And how an emoty value is handled?

Reply:
Ok, repeats can be a string ... this seems to be ok ... so still question is - for people that update the adapter ... what the value will be? undefined? or any of the provided values?